src/languages/kotlin.js

```diff
@@ -0,0 +1,105 @@
+/*
+ Language: Kotlin
+ Author: Sergey Mashkov <cy6erGn0m@gmail.com>
```

Comment:
Are there any rules to choose valid "category"? Is it fine to use "common" here?